stac-api/src/main/java/io/arlas/server/stac/api/StacSearchRESTService.java

```diff
@@ -196,25 +199,28 @@ However, in cases where the box spans the antimeridian the first value (west-mos
     @Path("/search")
     @POST
     @Consumes(MediaType.APPLICATION_JSON)
-    @Produces({ "application/geo+json", MediaType.TEXT_HTML, MediaType.APPLICATION_JSON })
-    @ApiOperation(
-            value = "Search STAC items with full-featured filtering.",
-            notes = "Retrieve items matching filters. Intended as the standard, full-featured query API.")
+    @Produces({ "application/geo+json" })
+    @Operation(
+            summary = "Search STAC items with full-featured filtering.",
+            description = "Retrieve items matching filters. Intended as the standard, full-featured query API.")
     @ApiResponses(value = {
-            @ApiResponse(code = 200, message = "A feature collection.", response = StacFeatureCollection.class,
-                    responseContainer = "StacFeatureCollection"),
-            @ApiResponse(code = 400, message = "Invalid query parameter.", response = Error.class),
-            @ApiResponse(code = 500, message = "Arlas Server Error.", response = Error.class)})
+            @ApiResponse(responseCode = "200", description = "A feature collection.",
+                    content = @Content(array = @ArraySchema(schema = @Schema(implementation = StacFeatureCollection.class)))),
```

Comment:
Returns `StacFeatureCollection.class` (not array)

Reply:
ok